docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -325,6 +325,48 @@ locations.
    mv dev-client.* secrets
    ```
 
+## Certificate Rotation
+
+This section documents the steps taken to rotate certificates in dev and INT subscriptions
+
+1. Generate new certificates like we did in the 2 sections above.
+
+2. Import newly generated certificates to keyvault by running `import_certs_secrets`. Note that this does not include firstparty
+
+3. The OpenVPN configuration file needs to be manually updated after generating new certificates at the moment. To achieve this, edit the `vpn-<region>.ovpn` file and add the vpn-client certificate and private key
```

Comment:
Let's break this out a bit more as we actually need to update the VPN deployment as well:

1. Update the Azure VPN Gateway configuration by adding in the new CA generated
2. Update the vpn-<region>.ovpn file with the VPN client certificate and private key. 

Note that these two are separate certificates used (ca is for VPN Gateway config, client is for `ovpn` file.